e2e-tests/auth.setup.ts

```diff
@@ -1,17 +1,20 @@
 import {
-  test as setup
+  test as setup, chromium
 } from '@playwright/test';
 
 const adminFile = 'playwright/.auth/admin.json';
 
 setup('authenticate as admin', async ({ page }) => {
 
-  await page.goto(`https://${process.env.HOST}/auth/realms/SHOGun/protocol/openid-connect/auth?`
-  + `client_id=shogun-client&redirect_uri=https%3A%2F%2F${process.env.HOST}%2Fclient%2F&state=`
-  + 'f9c5c223-1028-4496-8fdb-ae1da0f5daba&response_mode=fragment&response_type='
-  + 'code&scope=openid&nonce=6cd0da9a-b45c-4a9e-931d-99b6725a148e');
+  await page.goto(`https://${process.env.HOST}/auth/realms/SHOGun`
+    + '/protocol/openid-connect/auth?client_id=shogun-client'
+    + `&redirect_uri=https%3A%2F%2F${process.env.HOST}%2Fclient%2F%3FapplicationId%3D21&state=9a983abe-3b0c-`
+    + '41cb-9b7e-1d9120956959&response_mode=fragment&response_type'
+    + '=code&scope=openid&nonce=72884466-0535-4a24-8c15-9e7f14d88a65');
 
+  // @ts-ignore
   await page.getByLabel('Username or email').fill(`${process.env.ADMIN_LOGIN}`);
+  // @ts-ignore
```

Comment:
I think these might not be necessary, can you check if you still need the `@ts-ignore`?

Reply:
You are right, i have removed the ts-ignore!